### PR TITLE
Gas refund to dev balance

### PR DESCRIPTION
**Pull request by sasurobert**
Gas refund to dev balance
Delete process if error which was called twice
Added statistics prints for load code process in account adapter implementations.

### CODE REVIEW

**Review by SebastianMarian — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

File: data/state/accountsDB.go [hunk 1]
```diff
@@ -34,7 +69,8 @@ type AccountsDB struct {
 	entries      []JournalEntry
 	mutOp        sync.RWMutex
 
-	numCheckpoints uint32
+	numCheckpoints       uint32
+	loadCodeMeasurements loadingMeasurements
```
Comment (SebastianMarian) [inline data/state/accountsDB.go:73]: loadCodeMeasurements *loadingMeasurements ?

File: data/state/accountsDB.go [hunk 2]
```diff
@@ -70,6 +106,9 @@ func NewAccountsDB(
 		dataTries:              NewDataTriesHolder(),
 		obsoleteDataTrieHashes: make(map[string][][]byte),
 		numCheckpoints:         numCheckpoints,
+		loadCodeMeasurements: loadingMeasurements{
```
Comment (SebastianMarian) [inline data/state/accountsDB.go:109]: loadCodeMeasurements: &loadingMeasurements ?

File: data/state/accountsDB.go (hunk 1 above)
Comment (iulianpascalau) [reply data/state/accountsDB.go:73]: done

File: data/state/accountsDB.go (hunk 2 above)
Comment (iulianpascalau) [reply data/state/accountsDB.go:109]: done

**Review by SebastianMarian — DISMISSED** [review]

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by SebastianMarian — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by SebastianMarian — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
Awesome job!

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**